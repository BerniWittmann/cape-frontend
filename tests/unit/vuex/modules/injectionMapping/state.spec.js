import iMState from '@/vuex/modules/injectionMapping/state'

describe('Vuex', () => {
  describe('Modules', () => {
    describe('injectionMapping', () => {
      describe('State', () => {
        it('has an empty base state', () => {
          expect(iMState).toMatchSnapshot()
        })
      })
    })
  })
})
