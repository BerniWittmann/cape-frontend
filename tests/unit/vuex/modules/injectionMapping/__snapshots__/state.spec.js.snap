// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Vuex Modules injectionMapping State has an empty base state 1`] = `
Object {
  "injectionMappings": Array [],
}
`;
